ComSemApp/templates/ComSemApp/teacher/expression_form.html

```diff
@@ -79,6 +77,27 @@ <h4 class="text-muted m-b-10 font-14">
         </div>
     </div>
 
+    <div class="row">
+        <div class="col-12">
+            <div class="form-group">
+                <label for="youglish">YouGlish:</label>
+                <div class="container">
+                    <div class="input-group w-25">
+                        <input class=" form-control rounded-end m-1" type="text" id="wordInput" placeholder="Enter a word">
+                        <button class="btn btn-primary m-1" type="button" id="addButton">+</button>
+                    </div>
+                    <div class="d-flex flex-wrap" id="bubbleContainer"></div>
+                </div>
+                <style>
+                    .btn-bubble:hover {
+                        background-color: #DB1A1A;
+                        border-color: #DB1A1A; 
+                      }
```

Comment:
There is basically no good bootstrap way to do this from my research

Reply:
yeah... bootstrap is good at everything except the things it's bad at